milestones/MILESTONE_002.md

```diff
@@ -0,0 +1,39 @@
+# Milestone 2
+
+| Item             | Value           |
+| ---------------- |-----------------|
+| Codename         | `Call Me Maybe` |
+| Timebox delivery | Q3 2022         |
+
+## Status
+
+In-flight! ✈️
+
+## Summary
+
+Milestone 2 is about component-level minimum viability. It lets us 
+make demos and examples, and begins
+integration. This sets up our next phase: bringing these 
+components together into a cohesive 
+[Web5](https://developer.tbd.website/projects/web5) Platform. That in turn 
+will form the basis for the 
+[tbDEX](https://developer.tbd.website/projects/tbdex) application.
+
+## Goals
+
+* Deliver MVP APIs for Components:
+  * [`dwn-sdk-js`](https://github.com/TBD54566975/dwn-sdk-js)
+  * [`ssi-service`](https://github.com/TBD54566975/ssi-service)
+  * [`ssi-sdk`](https://github.com/TBD54566975/ssi-sdk)
+  * DID (Decentralized IDs) SDKs
```

Comment:
What exactly are these SDKs? Who should be working on these?

Reply:
@csuwildcat knew where - some in SSI repos and some in DWN. Links to put here would be great.

Reply:
It's the ION packages (at least for now)

Reply:
@csuwildcat Links I can put in there?